config/coreboot-t400-maximized-debug.config

```diff
@@ -0,0 +1,15 @@
+CONFIG_VENDOR_LENOVO=y
+CONFIG_CBFS_SIZE=0x7FCFFF
```

Comment:
From README notes, based on ifdtool -f output saved under layout.txt, where end-begin of BIOS REGION gives that value,.